browser/brave_rewards/rewards_tab_helper.cc

```diff
@@ -9,7 +9,7 @@
 #include "brave/components/brave_rewards/browser/rewards_service.h"
 #include "brave/components/ipfs/buildflags/buildflags.h"
 #include "chrome/browser/profiles/profile.h"
-#if !defined(OS_ANDROID)
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
Do we need to include "build/build_config.h" or something similar when referencing this?

Reply:
Yes, we should. I added it already on the places where it was not explicitly included but it seems I missed 
"quite a few" more, thanks for the catch. Fixed in https://github.com/brave/brave-core/pull/12655/commits/7d7f5e7554abbad3f8d1baaff7d991cbd520bf51